client/commands/v2/pysa_server.py

```diff
@@ -102,15 +169,22 @@ async def process_open_request(
         self, parameters: lsp.DidOpenTextDocumentParameters
     ) -> None:
         document_path = parameters.text_document.document_uri().to_file_path()
+        self.file_tracker[str(document_path)] = True
+        # await self.log_and_show_message_to_client(f"Tracking Files: {self.file_tracker}")
```

Comment:
We shouldn't be leaving any commented out logging in files we want to commit (apply throughout the PR)